pkg/gofr/gofr.go

```diff
@@ -309,87 +311,108 @@ func (a *App) initTracer() {
 
 	traceExporter := a.Config.Get("TRACE_EXPORTER")
 	tracerURL := a.Config.Get("TRACER_URL")
+	authHeader := a.Config.Get("TRACER_AUTH_KEY")
 
-	// deprecated : tracer_host and tracer_port is deprecated and will be removed in upcoming versions
+	// deprecated : tracer_host and tracer_port are deprecated and will be removed in upcoming versions.
 	tracerHost := a.Config.Get("TRACER_HOST")
 	tracerPort := a.Config.GetOrDefault("TRACER_PORT", "9411")
 
-	if tracerURL != "" && traceExporter == "" {
-		a.Logger().Error("missing TRACE_EXPORTER config, should be provided with TRACER_URL to enable tracing")
-		return
+	if tracerURL == "" && tracerHost != "" && traceExporter != traceExporterGoFr {
+		a.Logger().Warn("TRACER_HOST and TRACER_PORT are deprecated, use TRACER_URL instead")
+
+		tracerURL = fmt.Sprintf("%s:%s", tracerHost, tracerPort)
```

Comment:
Nothing to add as an URI?

Reply:
yeah this is for zipkin  the main brach only recive host and port build a http url to export to zipkin  ,  this enable tracer_url with https and also keep with host and port

Reply:
maybe call endpoint could be better or not